plugin/trino-hive-hadoop2/src/test/java/io/trino/plugin/hive/TestHiveThriftMetastoreWithS3.java

```diff
@@ -53,17 +56,11 @@ public class TestHiveThriftMetastoreWithS3
     private final Path hadoopCoreSiteXmlTempFile;
     private final AmazonS3 s3Client;
 
-    @Parameters({
-            "hive.hadoop2.s3.endpoint",
-            "hive.hadoop2.s3.awsAccessKey",
-            "hive.hadoop2.s3.awsSecretKey",
-            "hive.hadoop2.s3.writableBucket",
```

Comment:
alternatively convert these into fields whose value you set using `System.getProperty`.

(Or if we do add `SystemPropertyParameterResolver` then remove existing uses of `System.getProperty` in some other Junit migration commits)

Reply:
I think that this is more convenient  than calling getProperty in the fields.